ipn/ipnlocal/local.go

```diff
@@ -1715,6 +1736,51 @@ func (b *LocalBackend) Ping(ctx context.Context, ip netaddr.IP, pingType tailcfg
 	}
 }
 
+func peerForIP(nm *netmap.NetworkMap, ip netaddr.IP) (_ *tailcfg.Node, ok bool) {
```

Comment:
We should at some point just have a `map[netaddr.IP]*tailcfg.Node`

Reply:
Maybe on the netmap struct

Reply:
done

Reply:
Rather, I moved the method there for now. We can do the map later.